### ISSUE TITLE
sigtermHandler not handled on FreeBSD

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#471]
```
What version of Redis you are using, in what kind of Operating System?
2.2.1, FreeBSD 8.0

What is the problem you are experiencing?
Redis doesn't save rdb on disk when receives SIGTERM.

What steps will reproduce the problem?
kill -TERM redispid

Please provide any additional information below.
Move sigtermHandler out of #ifdef HAVE_BACKTRACE condition, because 
sigtermHandler doesn't use backtrace functionality.

Patch to master branch attached.
```

Original issue reported on code.google.com by `Konstant...@gmail.com` on 25 Feb 2011 at 8:12

Attachments:
- [redis-sigterm.patch](https://storage.googleapis.com/google-code-attachments/redis/issue-471/comment-0/redis-sigterm.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the patch. I've changed it a little to put all signal handling setup 
in a single function.

https://github.com/pietern/redis/tree/2.2-sigs
https://github.com/pietern/redis/tree/unstable-sigs
```

Original comment by `pcnoordh...@gmail.com` on 6 Mar 2011 at 4:52
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This is now available in both the 2.2 and unstable branches.
```

Original comment by `pcnoordh...@gmail.com` on 7 Mar 2011 at 11:34
- Changed state: **Fixed**
